contracts/0.8.9/oracle/ValidatorsExitBusOracle.sol

```diff
@@ -216,10 +219,13 @@ contract ValidatorsExitBusOracle is BaseOracle, PausableUntil {
     {
         _checkMsgSenderIsAllowedToSubmitData();
         _checkContractVersion(contractVersion);
+        bytes32 dataHash = keccak256(abi.encode(data.data));
```

Comment:
We should get `keccak256` from data + metadata (in our case it should be some type of data version or data fomat)

ToDo: @F4ever describe new data formats in docs.

Reply:
```suggestion
        bytes32 dataHash = keccak256(data.data);
```